src/components/Modal/Modal.tsx

```diff
@@ -39,7 +39,7 @@ const CloseButtonContainer = styled(Button.Container)`
   ${({ closeButtonAttachment }: { closeButtonAttachment: string }) => {
     let distance;
     let position;
-    let display_type = 'flex'; //what is the desired default for the close button's display-type??
+    let display = 'in-line flex'; //default display type
```

Comment:
This should be `inline-flex`, which is a display attribute
https://developer.mozilla.org/en-US/docs/Web/CSS/display
```suggestion
    let display = 'inline-flex'; //default display type
```